minishift-ci-index.yaml

```diff
@@ -26,7 +26,7 @@
                 credential-id: minishift_secret_password
```

Comment:
Minishift ? UGH

Reply:
This was added by the maintainer, we'll remove these from here any ways once we get the CRC namespace in centos ci